app/views/Licenses.tsx

```diff
@@ -61,28 +60,35 @@ export const LicensesScreen = ({
     <NavigationBarWrapper
       title={t('screen_titles.legal')}
       onBackPress={backToMain}>
-      <ScrollView contentContainerStyle={styles.contentContainer}>
+      <ScrollView
+        contentContainerStyle={styles.contentContainer}
+        alwaysBounceVertical={false}>
         <View>
-          <Typography style={styles.heading} use='headline2'>
-            {legalHeaderText}
-          </Typography>
-          <Typography style={styles.body} use='body1'>
-            {t('label.legal_page_address')}
-          </Typography>
-          <TouchableOpacity
-            onPress={handleOnPressOpenUrl('mailto:info@pathcheck.org')}>
-            <Typography style={styles.hyperlink}>{infoAddress}</Typography>
-          </TouchableOpacity>
-          <TouchableOpacity
-            onPress={handleOnPressOpenUrl('covidsafepaths.org')}>
-            <Typography style={styles.hyperlink}>{pathCheckAddress}</Typography>
-          </TouchableOpacity>
+          <Typography use='headline2'>{legalHeaderText}</Typography>
+          <View style={{ paddingTop: 10, paddingLeft: 24 }}>
```

Comment:
```suggestion
          <View style={{ paddingTop: 12, paddingLeft: 24 }}>
```
or better yet

```suggestion
          <View style={{ paddingTop: Spacing.xSmall, paddingLeft: Spacing.medium }}>
```

In general its best to stick to multiples of 4 for px, and to try and limit the number of spacing values used as it leads to a less visually noisy look and feel.

Reply:
💯